### ISSUE TITLE
Kiwix OPDS Feed Title

### ISSUE THREAD

**Issue by rgaudin** [kiwix/libkiwix#915]
Kiwix OPDS feed's title is `All Entries` which makes sense in the context of the `entries` endpoint within the API but when used as a Feed source (as advertise in the kiwix-serve viewer), it doesn't.

An `All Entries` subscription is not a good name.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:opds="https://specs.opds.io/opds-1.2"
      xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">
  <id>8cad36a0-64e2-7996-0472-00c09bc7ec79</id>

  <link rel="self"
        href="/catalog/v2/entries"
        type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
  <link rel="start"
        href="/catalog/v2/root.xml"
        type="application/atom+xml;profile=opds-catalog;kind=navigation"/>
  <link rel="up"
        href="/catalog/v2/root.xml"
        type="application/atom+xml;profile=opds-catalog;kind=navigation"/>

  <title>All Entries</title>
  <updated>2023-03-15T11:56:05Z</updated>
```

Ii might be difficult to solve, especially given a meaningful name might not be generic, so maybe we assume that all syndication readers allow renaming and users would go through that extra step.

**Comment by kelson42** [comment]
Thinking loud. The problem of kiwix-serve instance name/title is broader than just here. Welcome page title is a topic as well. I was thinking tha the problem shoykd be solved by a custom welcom page... but I wonder if we would still benefit of a --instanceTitle option?

**Comment by rgaudin** [comment]
It could also be part of some library-wide metadata because in this context, the library is the instance.

I can't recall the content of the discussion we had around categories/tags in Paris but I have the feeling it was somewhat connected.

Are there already other *static* information you can think of that you'd see user wanting to customize?

**Comment by kelson42** [comment]
@rgaudin Your last remark makes sense if we consider at some point to replace the custom library xml by a proper OPDS feed.

**Comment by kelson42** [comment]
@rgaudin At this stage, my opinion on this issue has slightly evolved:
* A better name should be found like "Kiwix Server library entries"
* We could consider that customisation should be done via reverse proxy if wanted (this is machine readable content)

**Comment by rgaudin** [comment]
> A better name should be found like "Kiwix Server library entries"

It's better but not that much

> We could consider that customisation should be done via reverse proxy if wanted (this is machine readable content)

Are you suggesting to close this as wontfix?
Is the request not valid? not worth the effort? not worth managing an additional data?

**Comment by kelson42** [comment]
> Are you suggesting to close this as wontfix?\nIs the request not valid? not worth the effort? not worth managing an additional data?

Yes, not really worth the effort... but I mostly want to challenge your wish to have this natively.

**Comment by rgaudin** [comment]
> but I mostly want to challenge your wish to have this natively.

Not convinced! Title is part of the Atom spec and is required for a `<feed />`. It's the main distinctive element of a Feed (Icon could be one indeed). If we're serious about this Atom feature, it has to be customizable.

I'd prefer we keep this open forever as I believe we'll eventually need instance metadata but we can close as well since this is not happening in the foreseeable future.
